### PR TITLE
[bash_script] Add mobilenetv2/squeezenet ncnn example

### PR DESCRIPTION
**Pull request by sharaelong**
Add a bash script example of object detection using mobilenetv2 and image classification using squeezenet with ncnn framework.

Self evaluation:

Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X] Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by myungjoo** [comment]
Please update https://github.com/nnstreamer/nnstreamer-example/blob/main/bash_script/example_models/get-model.sh so that the corresponding model files can be downloaded automatically.

**Comment by goranmoomin** [comment]
> Please update https://github.com/nnstreamer/nnstreamer-example/blob/main/bash_script/example_models/get-model.sh so that the corresponding model files can be downloaded automatically.

Done.

File: bash_script/example_image_classification_ncnn/gst-launch-image-classification-ncnn.sh
Comment (myungjoo) [inline bash_script/example_image_classification_ncnn/gst-launch-image-classification-ncnn.sh:1]: Empty?

Comment (chosanglyul) [reply bash_script/example_image_classification_ncnn/gst-launch-image-classification-ncnn.sh:1]: It looks like you're only looking at one commit. In that commit, the permissions of that file were changed from 644 to 755.

File: bash_script/example_models/get-model.sh [hunk 1]
```diff
@@ -50,6 +51,13 @@ elif [[ ${model} == "image-classification-caffe2" ]]; then
   wget ${download_url}/init_net.pb
   mv synset_words.txt labels.txt
 
+elif [[ ${model} == "image-classification-ncnn" ]]; then
+  mkdir -p ncnn_model
+  cd ncnn_model
+  wget "https://github.com/Tencent/ncnn/raw/master/examples/synset_words.txt"
```
Comment (chosanglyul) [inline bash_script/example_models/get-model.sh:57]: We found some issues in our script. We will fix it in a short time.

Comment (goranmoomin) [reply bash_script/example_models/get-model.sh:58]: This is fixed.

**Comment by goranmoomin** [comment]
We believe that after installing the relevant scrips, the following examples will work without any problems.

```
$ get-model.sh object-detection-ncnn && gst-launch-object-detection-ncnn.sh
$ get-model.sh image-classification-ncnn && gst-launch-image-classification-ncnn.sh
```

**Review by anyj0527 — APPROVED** [review]
COOL. I could do this example easily on my local machine. LGTM👍

File: bash_script/example_image_classification_ncnn/gst-launch-image-classification-ncnn.sh
Comment (myungjoo) [reply bash_script/example_image_classification_ncnn/gst-launch-image-classification-ncnn.sh:1]: Yes, actually, ideally, you need to do it in a commit adding ```gst-launch-image-classification-ncnn.sh```, not int a commit adding a download script. In this PR, I won't mind it, but please keep in mind that a single commit should have a single topic.

**Pull request merged**